src/components/Profile.tsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import { useState, useEffect } from 'react';
+import { BackendContext } from '../util/api';
+
+const useStyles = makeStyles(() => ({
```

Comment:
Is there a reason why we are specifying styles within the code? I am not opposed to it but why isnt this in a CSS file?

Reply:
I think each component should have their own css, because most of the time, the CSS of one component won't be needed for another component. Otherwise, if there's css for more general elements, it would be in the main CSS file. That's just my opinion, but most of the people wrote CSS in the component file.